### ISSUE TITLE
Change the configure to Azure GPT4, but the provision doesnt go throught

### ISSUE THREAD

**Issue by InvitedTea** [Azure-Samples/azure-search-openai-demo#726]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ *] bug report -> please search issues before submitting
- [ ] feature request
- [ *] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> for the main.bicep, I change the configuration: param chatGptModelName string = (openAiHost == 'azure') ? 'gpt-4' : 'gpt-4'
then run  azd up

### Any log messages given by the failure
>
ERROR: deployment failed: failing invoking action 'provision', error deploying infrastructure: deploying to subscription:

Deployment Error Details:
InvalidTemplateDeployment: The template deployment 'openai' is not valid according to the validation procedure. The tracking id is '3f2b6bab-caca-4f67-83e5-d9431ce6f3d0'. See inner errors for details.
SpecialFeatureOrQuotaIdRequired: The current subscription does not have feature required by this model 'Format: OpenAI, Name: gpt-4, Version: 0613' and SKU 'Standard'.

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### azd version?
> run `azd version` and copy paste here.

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pamelafox** [comment]
Does it work when you deploy to 3.5? I thought that gpt-4 was enabled for everyone who had 3.5 enabled.

**Comment by InvitedTea** [comment]
Yes. I have deployed with 3.5 without any issue. It works well. When I change it to gpt4., it fail the provision.

**Comment by pamelafox** [comment]
Okay, can you confirm if you're using a new resource group or an existing resource group?

**Comment by InvitedTea** [comment]
Yes I use a brand new resource group

**Comment by pamelafox** [comment]
Hm. We're stumped. Is there a place in the Portal where you can log a support request? Sometimes they have more insights than we do.

**Comment by InvitedTea** [comment]
Thank @pamelafox , can you share the formal main.bicep example how we expect to change gpt4. that will give me a clear sense. Thank you

**Comment by pamelafox** [comment]
@InvitedTea Were you able to provision it? I haven't heard issues from others recently re GPT 4 in new resource groups.

**Comment by ihmunro** [comment]
I am getting a different error

The subscription does not have QuotaId/Feature required by SKU 'S0' from kind 'OpenAI' or contains blocked QuotaId/Feature. (Code: SpecialFeatureOrQuotaIdRequired)
